src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/ReferenceAssemblies.cs

```diff
@@ -739,8 +755,121 @@ public static class NetStandard
                     new PackageIdentity(
                         "NETStandard.Library",
                         NuGetVersion.Parse("2.0.3")),
-                    @"build\netstandard2.0\ref")
-                .AddAssemblies(ImmutableArray.Create("netstandard"));
+                    Path.Combine("build", "netstandard2.0", "ref"))
+                .AddAssemblies(ImmutableArray.Create("netstandard"))
+                .AddAssemblies(ImmutableArray.Create(
+                    "Microsoft.Win32.Primitives",
```

Comment:
Is this the only reasonable way to handle this? Will this be scalable?

Reply:
It's hard-coded in the NuGet package using a custom targets file, rather than using the standard patterns.